package_manifest.yaml

```diff
@@ -0,0 +1,382 @@
+    nodejs-axios: {}
+    nodejs-babel-cli: {}
+    nodejs-babel-core: {}
+    nodejs-babel-loader: {}
+    nodejs-babel-plugin-lodash: {}
+    nodejs-babel-plugin-transform-class-properties: {}
+    nodejs-babel-plugin-transform-object-assign: {}
+    nodejs-babel-plugin-transform-object-rest-spread: {}
+    nodejs-babel-polyfill: {}
+    nodejs-babel-preset-env: {}
+    nodejs-babel-preset-react: {}
+    nodejs-babel-register: {}
+    nodejs-brace: {}
+    nodejs-c3: {}
+    nodejs-change-emitter: {}
+    nodejs-classnames: {}
+    nodejs-compression-webpack-plugin: {}
+    nodejs-css-loader: {}
+    nodejs-datatables.net: {}
+    nodejs-datatables.net-bs: {}
+    nodejs-debug: {}
+    nodejs-diff: {}
+    nodejs-dotenv: {}
+    nodejs-expose-loader: {}
+    nodejs-extract-text-webpack-plugin: {}
+    nodejs-fbjs: {}
+    nodejs-file-loader: {}
+    nodejs-follow-redirects: {}
+    nodejs-harmony-reflect: {}
+    nodejs-hoist-non-react-statics: {}
+    nodejs-identity-obj-proxy: {}
+    nodejs-ipaddr.js: {}
+    nodejs-is-buffer: {}
+    nodejs-isomorphic-fetch: {}
+    nodejs-jquery: {}
+    nodejs-jquery-flot: {}
+    nodejs-jquery-ujs: {}
+    nodejs-jquery.cookie: {}
+    nodejs-js-tokens: {}
+    nodejs-jsdom: {}
+    nodejs-jstz: {}
+    nodejs-lodash: {}
+    nodejs-lodash-webpack-plugin: {}
+    nodejs-loose-envify: {}
+    nodejs-ms: {}
+    nodejs-multiselect: {}
+    nodejs-node-gyp: {}
+    nodejs-node-sass: {}
+    nodejs-object-assign: {}
+    nodejs-patternfly: {}
+    nodejs-patternfly-react: {}
+    nodejs-performance-now: {}
+    nodejs-prop-types: {}
+    nodejs-raf: {}
+    nodejs-react: {}
+    nodejs-react-bootstrap: {}
+    nodejs-react-bootstrap-switch: {}
+    nodejs-react-c3js: {}
+    nodejs-react-dom: {}
+    nodejs-react-ellipsis-with-tooltip: {}
+    nodejs-react-fontawesome: {}
+    nodejs-react-numeric-input: {}
+    nodejs-react-onclickoutside: {}
+    nodejs-react-password-strength: {}
+    nodejs-react-redux: {}
+    nodejs-recompose: {}
+    nodejs-redux: {}
+    nodejs-redux-form: {}
+    nodejs-redux-form-validators: {}
+    nodejs-redux-logger: {}
+    nodejs-redux-thunk: {}
+    nodejs-sass-loader: {}
+    nodejs-seamless-immutable: {}
+    nodejs-select2: {}
+    nodejs-style-loader: {}
+    nodejs-symbol-observable: {}
+    nodejs-urijs: {}
+    nodejs-url-loader: {}
+    nodejs-uuid: {}
+    nodejs-webpack: {}
+    nodejs-webpack-stats-plugin: {}
+    nodejs-zxcvbn: {}
+    puppet-agent-oauth: {}
+    puppet-agent-puppet-strings: {}
+    puppet-agent-yard: {}
+    rubygem-foreman_api: {}
+    rubygem-foreman_maintain: {}
+    rubygem-kafo: {}
+    rubygem-kafo_parsers: {}
+    rubygem-kafo_wizards: {}
+    rubygem-rb-inotify: {}
+    rubygem-rsec: {}
+    rubygem-rubyipmi: {}
+katello:
+  vars:
+    package_base_dir: 'packages/katello/'
+    releasers:
+    - koji-katello
+  hosts:
+    katello: {}
+    katello-certs-tools: {}
+    katello-client-bootstrap: {}
+    katello-host-tools:
```

Comment:
do we want a `katello-client` group for `katello-host-tools` and `katello-repos` (why is that built into client anyways?)?

@ehelms what do you think?

Reply:
Hrmmm, since they are their own repositories -- I think that makes sense.

The `katello-repos` RPM has both server repositories and client repositories contained within it. So we build it for all of the buildroots to get the client sub-package into the client repos.

Reply:
@pcreech Want to make this small group? Then I think this is good to go.

Reply:
@ehelms done